docs/Installing.asciidoc

```diff
@@ -64,15 +64,14 @@ To add the development repository to your system, you can use these commands.
 zypper ar -f obs://devel:openQA/openSUSE_Tumbleweed devel-openQA
 
 
-# openSUSE Leap 15
-zypper addrepo https://download.opensuse.org/repositories/devel:/openQA/openSUSE_Leap_15.0/ devel-openQA
-
-
-# openSUSE Leap 42.3
-zypper ar -f obs://devel:openQA/openSUSE_Leap_42.3 devel-openQA
-zypper ar -f obs://devel:openQA:Leap:42.3/openSUSE_Leap_42.3 devel-openQA-perl-modules
+# openSUSE Leap <version>
+zypper ar -f obs://devel:openQA/openSUSE_Leap_<version> devel-openQA
+zypper ar -f obs://devel:openQA:Leap:<version>/openSUSE_Leap_<version> devel-openQA-perl-modules
```

Comment:
you're not serious here. it's not even a full hour since you complained about missing deps on just devel:openQA

Reply:
I wrote this with hard coded version because newcomers were typing the command wrong and asking in IRC.
I am against this change.

Reply:
@coolo Looks similar to my version :-)

Reply:
But, what is the reason of using obs url instead of https://download...?

I remember having problems with the obs url the first time I edit this doc

Reply:
it's shorter and more obvious?

Reply:
updated